### ISSUE TITLE
Cannot write hashtags into pinned post fields 

### ISSUE THREAD

**Issue by aproposnix** [tootsuite/mastodon#15410]
### Expected behaviour
As a user, I would like to be able to use my own hashtags for pinned posts so that I can follow streams of interest to me. 

### Actual behaviour
When I try to add some hashtags manually to a pinned post (for example, #antifa or #trump) I am only provided with a list of similar hashtags and not able to use the hashtag I tried to enter (#antifa in this case)

### Steps to reproduce the problem

Steps to reproduce (assuming the user has a pinned column already) 
1. On the pinned column, click on the setting's icon (upper right) 
2. Toggle on "Include additional tags for this column"
3. in the "Any of these" field enter #antifa
4. You are given a list oof tags which doesn't include your tag and you cannot manually add it. 

### Specifications

I'm on mastodon.social

Some good Samaritan did some triage for me where he describes the issue: 
https://mastodon.social/@sparr/105425554237165801

>I think this is the relevant code:
https://github.com/tootsuite/mastodon/blob/a549415868fe23e0afaf258c17afafac117d0163/app/javascript/mastodon/features/hashtag_timeline/components/column_settings.js#L62-L75

>AsyncSelect does not allow freeform text input, it only allows selecting things that the search returns. This control should either be changed to one allowing freeform entry, or the back end could always return the entered text as a result.

**Comment by Danie10** [comment]
Actually it is worse for me. I deleted my hashtags to watch as they were suddenly not displaying properly, although filter was working. I cannot add them back again due to same/similar issue. I tested it with known and even trending hashtags, it says "no suggestions found" and removed the hashtag. Tried different browsers and even a Tor browser without any extensions.

It certainly worked for me months back when I set this filter. I rely heavily on such a filter pinned in a column to track discussions on topics for people I don't follow.

**Comment by safkhet** [comment]
I'm having the same problem as @Danie10 . I'm on mastodon.social. Not sure how long ago it started, but at least a couple of weeks.

**Comment by miKa-pyon** [comment]
This may need a separate issue but I have the same problem as the two above!
I started using Mastodon in the first week of October. Adding extra tags to pinned columns was working.
Now for about ~2 weeks I noticed I am not able to enter anything in the hashtag search field on the pinned column.
The existing columns with extra hashtags are still working but the extra hashtags are shown as empty under a columns "Show Settings → Any of these" while the column header still displays them.

**Comment by Danie10** [comment]
@miKa-pyon thanks for confirming! I'd given up checking. Tested now and my column works fine again. Really happy about this.

**Comment by miKa-pyon** [comment]
Well so much for that, all my columns are broken again.
Just like described above:
`<div class=tags-12jo7m5 column-select__multi-value__label>` has no text.

I suspect it happens after I log out, restart the webbrowser and log in again.
Cannot believe not more people notice this.

**Comment by Danie10** [comment]
Yes, I've not logged out so has maintained the filter for nearly two weeks now.

**Comment by lrmikestone** [comment]
Just checking on this issue. We're on 3.4.4, and we're still seeing this. I can't even flip the toggle to Include additional tags for this column. It just reverts back to "Off" as soon as I close the configuration option.

**Comment by FediVideos** [comment]
Had a report from a new user saying they are having this issue, they're on mstdn.social which is running 3.5.2glitch. They gave as an example the tag "disability". (Post is here: https://mstdn.social/@neurodivergentgeek/108261025588500103)

**Comment by aproposnix** [comment]
Just want to pop in to say that this is getting way worse for me since there's been an increase in users. It's at the point right now where I cannot create pinned columns with my own hashtags (unless they happen to appear on the top of the list which is rare).
